eip-1271/nodejs/src/index.ts

```diff
@@ -0,0 +1,224 @@
+import { LitContracts } from "@lit-protocol/contracts-sdk";
+import {
+  AuthMethodScope,
+  AuthMethodType,
+  LitNetwork,
+  LIT_RPC,
+} from "@lit-protocol/constants";
+import { LitNodeClient } from "@lit-protocol/lit-node-client";
+import ethers from "ethers";
```

Comment:
Wouldn't work unless:

```suggestion
import * as ethers from "ethers";
```

Reply:
Changing it to `import { ethers } from "ethers";` not sure why the other way was giving you an error though